src/utilities/text/text.scss

```diff
@@ -7,3 +7,7 @@
 .u-text-action {
   color: var(--theme-color-text-action) !important;
 }
+
+.u-text-align-center {
```

Comment:
Is the word `align` necessary?

Reply:
I have no preference. 🙃 

I will rename it to `u-text-center`. 😉

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1771/commits/d4bb66380290358694a6c2178da3c85ada6c72c7